www/js/metrics.js

```diff
@@ -443,6 +456,11 @@ angular.module('emission.main.metrics',['nvd3', 'emission.services', 'ionic-date
           var aggCount = results[2].aggregate_metrics;
```

Comment:
This checks if the data is empty and if its empty non of the calculation will be triggered. The calculation will only run once. In the else case `var userDuration = results[0].user_metrics;` is empty but it does't matter because if its empty the UI will say No Summary Data. In line 615 `first` is set to `false` after 1st run. So now whenever the user selects costume data `var userDuration = results[0].user_metrics;` in the else case is used. Both initial call for last week data and user's costume call have same vars but twoWeeksAgo** is only set in first case.